packages/app/src/cli/metadata.ts

```diff
@@ -1,5 +1,5 @@
 import {metadata as metadataLib, monorail} from '@shopify/cli-kit'
-import type {PickByPrefix} from '@shopify/cli-kit/typing/pick-by-prefix'
+import type {PickByPrefix} from '@shopify/cli-kit/common/typing/pick-by-prefix'
```

Comment:
`typing` or `types` is usually used to to group declaration files for modules that don't have types. What about naming this one `@shopify/cli-kit/common/ts` to reflect that are Typescript utilities?